### PR TITLE
Nodes should reject decryption requests for expired rituals

### PR DESCRIPTION
**Pull request by derekpierre**
**Type of PR:**
- [ ] Bugfix
- [X] Feature
- [ ] Documentation
- [ ] Other

**Required reviews:** 
- [ ] 1
- [X] 2
- [ ] 3

**What this does:**
Expired rituals should not be allowed by nodes.

Technically this was implemented at the smart contract level for defining duration, but the nodes never enforced it. It's a bug from a testnet perspective, but a feature for 7.0.0 that wasn't completed previously, hence the newsfragment is a "feature" and not "bug".

**Issues fixed/closed:**
> - Fixes #...

**Why it's needed:**
Otherwise, what's the point of paying for longer periods of time 😅 .

**Notes for reviewers:**
> What should reviewers focus on? 
> Is there a particular commit/function/section of your PR that requires more attention from reviewers?

### CODE REVIEW

**Review by manumonti — APPROVED** [review]
LGTM! Good catch! 🎣

File: examples/testnet_simple_taco.py [hunk 1]
```diff
@@ -38,7 +38,7 @@
     blockchain_endpoint=polygon_endpoint,
     registry=registry,
 )
-ritual_id = 1  # got this from a side channel
+ritual_id = 3  # got this from a side channel
 ritual = coordinator_agent.get_ritual(ritual_id)
 
 # known already authorized signer for ritual 1
```
Comment (cygnusv) [inline examples/testnet_simple_taco.py:44]: This mentions ritual 1. But more importantly, is this hardcoded Enrico also authorized for ritual 3?

**Review by cygnusv — APPROVED** [review]
Minor RFC, but apart from that, all good!

File: examples/testnet_simple_taco.py (hunk 1 above)
Comment (derekpierre) [reply examples/testnet_simple_taco.py]: ✔️ - updated.

> is this hardcoded Enrico also authorized for ritual 3?

It is. I used the `nucypher_dkg` script to authorize the hardcoded Enrico while testing ritual 3 - https://github.com/nucypher/nucypher/blob/development/scripts/hooks/nucypher_dkg.py#L336

**Pull request merged**